### PR TITLE
[MODULAR] Moves techweb all_nodes.dm edits to modular

### PR DESCRIPTION
**Pull request by vinylspiders**
## About The Pull Request

This does some housekeeping and moves all of the Skyrat edits out of `all_nodes.dm` and into `master_files`.

A couple of these edits were alphabetized and unfortunately no longer will be as a consequence of this. 

We could sort them after the fact if we wanted to but for now considering how inconsistent it was (most of them were not alphabetized) I'm just going to keep the Skyrat additions at the end of the list. 

That is how the majority of them were before, anyway, and it makes things more consistent in that way.

## How This Contributes To The Skyrat Roleplay Experience

Easier time for maintainers. Less chances of merge errors too, and IMO the more examples of modular code we have the better the chances of coders following suit. Plus, I think having all the modular edits laid out in one place like this is just preferable, personally. It's way easier to see them all at a glance this way.

## Proof of Testing

<details>
<summary>You can see all the techweb datums loaded without errors and the designs are there</summary>

image

(Disregard bowtie related runtime, my char's prefs are from the future)

</details>

## Changelog

:cl:
refactor: moves ALL of the skyrat edits out of modules/research/techweb/all_nodes.dm and into modular overrides
/:cl:

### CODE REVIEW

**Comment by softcerv** [comment]
Great PR, I wonder if there is any documentation we might want to update for this.

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -1988,7 +1898,7 @@
 		"savannah_ivanov_targ",
 		"savannah_ivanov_torso",
 	)
-	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
```
Comment (softcerv) [inline code/modules/research/techweb/all_nodes.dm:1901]: If you don't mind me asking, why is this being changed?

File: modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm [hunk 2]
```diff
@@ -15,8 +14,47 @@
...
+		"synth_lungs",
+		"synth_stomach",
+		"synth_ears",
+		"synth_heart",
+	)
+
+/////////////////////////Biotech/////////////////////////
+
+/datum/techweb_node/adv_biotech/New()
+	. = . = ..()
```
Comment (softcerv) [inline modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm]:
```suggestion
	. = ..()
```
I think you made an oopsie

File: modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm [hunk 3]
```diff
@@ -29,8 +67,166 @@
...
+
+/////////////////////////EMP tech/////////////////////////
+
+/datum/techweb_node/emp_basic/New()
+	. = ..()
+	design_ids += list(
+		"gownmedicell",
+		"bedmedicell",
+	)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
```
Comment (softcerv) [inline modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm]: Why is this included here?

File: modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm [hunk 4]
```diff
@@ -42,5 +238,10 @@
...
 
-// MODULAR MODIFICATIONS
-//
+/////////////////////////robotics tech/////////////////////////
+
+/datum/techweb_node/cyborg_upg_med/New()
+	. = ..()
+	design_ids -= list(
+		"borg_upgrade_pinpointer",
+	)
```
Comment (softcerv) [inline modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm]: I think you might want to combine this with the `/datum/techweb_node/cyborg_upg_med/New()` earlier in this file

**Review by softcerv — COMMENTED** [review]
some stuff I noticed

File: code/modules/research/techweb/all_nodes.dm (hunk 1 above)
Comment (vinylspiders) [reply code/modules/research/techweb/all_nodes.dm:1901]: Oh, that's how it is upstream. There were a few diffs that were not marked as skyrat edits.

This file has been matched upstream such that there are no more diffs (aka the point of this pr).

File: modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm (hunk 2 above)
Comment (vinylspiders) [reply modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm]: Lol. The funny thing is this works. But yes definitely oopsie

File: modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm (hunk 3 above)
Comment (vinylspiders) [reply modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm]: I've no clue. Must have been a copy paste I missed.

File: code/modules/research/techweb/all_nodes.dm (hunk 1 above)
Comment (softcerv) [reply code/modules/research/techweb/all_nodes.dm:1901]: oh wow lol

File: modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm (hunk 4 above)
Comment (vinylspiders) [reply modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm]: The idea was to keep removals and additions in separate spots. I dunno, might be over organizing on my part. But it made sense in my head.

**Comment by vinylspiders** [comment]
> some stuff I noticed

Nice, thanks for catching those.

File: modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm (hunk 4 above)
Comment (vinylspiders) [reply modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm]: K, sections are now condensed.

**Review by GoldenAlpharex — APPROVED** [review]
Looks good!
